src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -166,7 +166,9 @@ public static IEnumerable<string> LanguagesForWorker(WorkerRuntime worker)
 
         public static WorkerRuntime GetCurrentWorkerRuntimeLanguage(ISecretsManager secretsManager)
         {
-            var setting = secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals(Constants.FunctionsWorkerRuntime, StringComparison.OrdinalIgnoreCase)).Value;
+            var setting = secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals(Constants.FunctionsWorkerRuntime, StringComparison.OrdinalIgnoreCase)).Value
```

Comment:
Does the order matter here? Should we check env variable first?

Reply:
Same note as above